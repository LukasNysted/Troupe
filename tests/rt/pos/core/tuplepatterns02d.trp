let fun f () = (1,3,7) in
  case f () of
    (1,5,7) => false
  end