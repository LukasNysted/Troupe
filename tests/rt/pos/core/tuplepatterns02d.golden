2024-02-14T11:33:56.513Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread 5cb562cf-30ca-4a3d-aeda-16805f13a610@{}%{}
>> pattern match failure in case expression at :2:3
